### ISSUE TITLE
Link to /kb/ somewhere explicitly

### ISSUE THREAD

**Issue by sknebel** [bgptools/issues#54]
URL:
```
https://bgp.tools/kb/
```

Browser (User-Agent if you can): 
```
n/a
```

Error base64 blob (If a site error):
```
n/a
```

What were you expecting:
```
This useful page to be linked somewhere, e.g. in the navigation at the page bottom
```

What happened:
```
I only found it by editing the URL after I noticed the /kb/ prefix 
```

**Comment by athompson-merlin** [comment]
Same... just a link in the footer to "Knowledge Base" would be helpful.  I had to resort to Google to find the page talking about SSH'ing into the Super-LG.

**Comment by benjojo** [comment]
All KB pages now have a link at the bottom to the index

<img width="1159" height="489" alt="Image" src="https://github.com/user-attachments/assets/37641c60-bf14-4179-be23-31c69dedf0de" />

**Comment by athompson-merlin** [comment]
Er, sorry, while that's a very good idea, the initial problem wasn't navigating within the KB, the problem was discovering that the KB exists in the first place.

I believe the easiest solution to that would be to add a "Knowlege Base" link to the bottom of the standard footer:

<img width="781" height="91" alt="Image" src="https://github.com/user-attachments/assets/11d69652-4a62-42a7-940a-e39ff3873514" />

**Comment by benjojo** [comment]
That makes sense on desktop, but looks kinda naff on mobile, I need to think more about how to better handle the footer on mobile devices before I do this

before
<img width="1080" height="2400" alt="Image" src="https://github.com/user-attachments/assets/045a850e-feb8-4bda-9f19-2284ee90a74c" />

after
<img width="1080" height="2400" alt="Image" src="https://github.com/user-attachments/assets/e85283c9-cae9-48bb-9222-766928c19b12" />

**Comment by athompson-merlin** [comment]
I see what you mean!

It doesn't _have_ to be in the footer - it could just be a link on the main page.  The problem is just there's no way to get to the KB (or discover it exists) unless you already know the URL.  I think there are some specific KB links buried in various customer profile pages, but a general user would have no way of discovering that trove of knowledge (that I know of, anyway).

**Comment by benjojo** [comment]
Yeah I get that,

Right now I actually don't actually think that the knowledge base is
that useful (at least from my point of view), it was thrown together
in a hurry for things like the changelog (which really should be it's
own part of the website and not just glued onto the side of the
knowledge base system)

There's definitely a large overhaul that as due here, but thanks to
the existence of search engines I don't think this is a huge priority
(as people can presumable search for things there) until I have a
large amount of content ready to go for this, I will however make a
ticket in my internal tracker for this so that this conversation is
not entirely lost to time

**Comment by athompson-merlin** [comment]
Makes sense.  FWIW, from my point of view, the KB has forestalled at least 4 or 5 emails to you so far.  YMMV.  TTYL.

Adam Thompson
Consultant, Infrastructure Services
***@***.***
100 - 135 Innovation Drive
Winnipeg, MB R3T 6A8
(204) 977-6824 or 1-800-430-6404 (MB only)
https://www.merlin.mb.ca<https://www.merlin.mb.ca/>
***@***.***Chat with me on ***@***.***>
